src/utils.js

```diff
@@ -28,18 +43,31 @@ function isGlob (candidate) {
 }
 
 /**
- * Take a glob; read the files. Return a Promise that ultimately resolves
- * to an Array of objects:
- * [{ path: original filepath,
- *   contents: utf-8 file contents}...]
+ * Take a glob; read the files, optionally running a `contentFn` over
+ * the contents of the file.
+ *
+ * @param {glob} glob of files to read
+ * @param {Object} Options:
+ *  - {Function} contentFn(content, path): optional function to run over content
+ * in files; defaults to a no-op
+ *  - {String} encoding
+ *
+ * @return {Promise} resolving to Array of Objects:
+ *  - {String} path
+ *  - {String || Mixed} contents: contents of file after contentFn
  */
-function readFiles (glob) {
+function readFiles (glob, {
```

Comment:
readFiles expanded to take optional `contentFn` to run over file contents before returning.